### ISSUE TITLE
Only getting archers in Barracks?

### ISSUE THREAD

**Issue by Paril** [ldtteam/minecolonies#2930]
### Minecolonies version

Version: minecolonies-1.12.2-0.9.3-RELEASE-universal.jar

### Expected behavior

Barracks assigns Archer or Swordsman-type guard at random

### Actual behaviour

Barracks only gives me archers

### Steps to reproduce the problem

I have a Barracks leveled up to level 4, with all towers upgraded to level 2. Each of them have only ever netted me archers. No idea if this is just super bad luck or intentional (should be able to change type at the tower to be honest...)

**Comment by Raycoms** [comment]
It's fairly random but you can assign them manually on the second page of
the tower as well. To either archer or knight

**Comment by Paril** [comment]
Hmm. Okay, so I found the manual reassignment, got that working. Weirdly enough, when I got the barracks to re-build the tower blocks (I broke and repaired for two of them), it originally said Knight in the UI, but changed to Ranger once it was completed again, so again I was stuck with all rangers till I found the re-assignment option. Maybe it should be a global flip-flop instead of 50% chance, because all of my coin flips were ranger, haha.
